pages/dkp/konvoy/2.2/image-builder/create-azure-image/index.md

```diff
@@ -80,7 +81,7 @@ Extract the bundle and `cd` into the extracted `konvoy-image-bundle-$VERSION_$OS
 Run the `konvoy-image` command to build and validate the image.
 
 ```sh
-konvoy-image build azure --client-id <azure_client_id> --tenant-id <azure_tenant_id> --overrides override-source-image.yaml images/azure/centos-7.yaml
+konvoy-image build azure --client-id "<azure_client_id>" --tenant-id <azure_tenant_id> images/azure/image.yaml --overrides overrides/image.yaml
 ```
```

Comment:
I believe on of the things Galina noticed was that these should be using the ```bash markdown because that gives you a little 'copy/paste' icon to click on when they are rendered.  ```sh does not.   Also, she suggested we add double quotes around both the <azure_client_id> and the <azure_tenant_id>.

So, I believe this is what she was suggesting:

```bash
konvoy-image build azure --client-id "<azure_client_id>" --tenant-id "<azure_tenant_id>" images/azure/image.yaml --overrides overrides/image.yaml
```

Also , there is another instance of this command on lines 89/90 that would also need these same change (add double quotes, change ```sh->```bash

Reply:
Got it... been spoiled by Confluence.  HA!  Made those Changs as well.